client/app/bundles/course/experience-points/components/ExperiencePointsTableRow.tsx

```diff
@@ -0,0 +1,196 @@
+import { FC, memo, useEffect, useState } from 'react';
+import { TableCell, TableRow } from '@mui/material';
+import equal from 'fast-deep-equal';
+import {
+  ExperiencePointsRecordMiniEntity,
+  ExperiencePointsRowData,
+} from 'types/course/experiencePointsRecords';
+
+import PointManagementButtons from 'course/users/components/buttons/PointManagementButtons';
+import NumberTextField from 'lib/components/core/fields/NumberTextField';
+import TextField from 'lib/components/core/fields/TextField';
+import Link from 'lib/components/core/Link';
+import { formatMiniDateTime } from 'lib/moment';
+
+interface Props {
+  id: number;
```

Comment:
Now i realise, i think this one can be retrieved from record as well. Can help to remove this?